kdc/kdc_locl.h

```diff
@@ -44,23 +44,57 @@ typedef struct pk_client_params pk_client_params;
 struct DigestREQ;
 struct Kx509Request;
 typedef struct kdc_request_desc *kdc_request_t;
+typedef struct astgs_request_desc *astgs_request_t;
```

Comment:
Because the definitions are required in the prototypes in kdc-proto.h and kdc-private.h. kdc_locl.h defines it. Maybe we can take the typedef out of kdc_locl.

Reply:
I'd prefer to see these in one place, if possible.  Your call as to how to do that...